src/routes/index.ts

```diff
@@ -308,4 +309,15 @@ router.route(config.UPDATE_CHECK_YOUR_ANSWERS_URL)
   .get(updateCheckYourAnswers.get)
   .post(updateCheckYourAnswers.post);
 
+router.route(config.UPDATE_BENEFICIAL_OWNER_GOV_URL)
```

Comment:
while you are at it, might as well update this for the register journey route methods too!